Cargo.toml

```diff
@@ -60,6 +60,23 @@ opt-level = "s"
 unused_qualifications = "warn"
 unsafe_op_in_unsafe_fn = "warn"
 
+# Lints for upgrading to the 2024 edition
+boxed_slice_into_iter = "warn"
+dependency_on_unit_never_type_fallback = "warn"
+deprecated_safe_2024 = "warn"
+edition_2024_expr_fragment_specifier = "allow" # ALLOW
+if_let_rescope = "allow" # ALLOW
+impl_trait_overcaptures = "warn"
+keyword_idents_2024 = "warn"
+missing_unsafe_on_extern = "warn"
+never_type_fallback_flowing_into_unsafe = "warn"
+rust_2024_guarded_string_incompatible_syntax = "warn"
+rust_2024_incompatible_pat = "warn"
+rust_2024_prelude_collisions = "warn"
+static_mut_refs = "warn"
+tail_expr_drop_order = "allow" # ALLOW
```

Comment:
`ALLOW`ed lints are mostly noise (probably)

unfortunately enabling the `rust-2024-compatibility` lint group, then `= "allow"`ing specific lints doesn't seem to work, for some reason

Reply:
https://doc.rust-lang.org/nightly/rustc/lints/groups.html

Reply:
Ok apparently I should've used `rust_2024_compatibility = { priority = -1, level = "warn" }`